libs/security-core/src/main/java/no/nav/testnav/libs/securitycore/domain/Token.java

```diff
@@ -10,7 +10,8 @@
 @Builder
 @AllArgsConstructor
 public class Token {
-    String value;
+    String accessTokenValue;
+    String refreshTokenValue;
```

Comment:
Tror denne klassen blir brukt i libs serverletsecurity også. Så tror du må enten oppdatere serverletsecurity eller lage to forskjellige Token klasser.